tests/FINDOLOGIC/Export/Tests/DataHelperTest.php

```diff
@@ -47,4 +56,49 @@ public function emptyValueProvider()
             'false' => [false, true]
         ];
     }
+
+    /**
+     * @dataProvider numericValueProvider
+     *
+     * @param $value string|int value that should be checked.
+     * @param $shouldCauseException bool should an exception be caused by given parameter.
+     */
+    public function testNumericValuesAreValidated($value, $shouldCauseException)
+    {
+        try {
+            $price = new UsergroupAwareNumericValue('dummies', 'dummy');
```

Comment:
The name of the variable is not correct anymore. Use e.g. `$numericValueElement`

Reply:
✅